Update README.md, proton.sk and electron.sk

sapphyrus opened the pull request:
Made most of the commands more compact and easier to understand, removed a lot of spaghetti code, added permission to colored signs (proton.coloredsign). All changes are untested but should work.

Vehicola commented:
Has this branch/new code been tested by you? I would like to know before merging, and what addons you used

sapphyrus commented:
Its untested but I can test it later.
Addons should've stayed the same since I mostly removed code. (I replaced most of the [<player>] arguments with [<player=%player%>] and removed the code that ran if the argument wasn't set, but take a look at it yourself
